superbuild/CMakeLists.txt

```diff
@@ -55,7 +55,19 @@ list(APPEND external_projects
   DCMTK 
   QtDCM 
   dtk
-  )
+  jsoncons
+  glew
+  asio
+  websocketpp
+  eigen
+  xcom 
+  gvirtualXRay
+  qwt 
+  mmg 
+  tetgen
+  #poisson
```

Comment:
you told me openssl is used, but not poisson for now, so it could be removed perhaps

Reply:
Poisson will be used soon but surface reconstruction (but it doen't have high priority). 
For those who want to compile SRC plugin, they have to uncomment ssl and compile with -j 1. It is better to comment it compile with -j10 (without ssl) then compile with -j 1 with ssl. 
I hope to find a better solution next time.